src/igvfd/schemas/open_reading_frame.json

```diff
@@ -0,0 +1,138 @@
+            "uniqueKey": true,
+            "submissionExample": {
+                "appscript": "CCSBORF1234",
+                "igvf_utils": "CCSBORF1234"
+            }
+        },
+        "gene": {
+            "title": "ENSEMBL GeneID",
+            "description": "ENSEMBL GeneID of official nomenclature approved gene. The GeneID does not include the current version number suffix.",
+            "type": "array",
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+                "title": "ENSEMBL GeneID",
+                "description": "An identifier of the Gene.",
+                "linkTo": "Gene",
+                "type": "string"
+            },
+            "submissionExample": {
+                "appscript": "[\"/genes/ENSG00000231542/\", \"ENSG00000264176\", \"/genes/ENSG00000263690/\"]",
+                "igvf_utils": "/genes/ENSG00000231542/, ENSG00000264176, /genes/ENSG00000263690/"
+            }
+        },
+        "protein_id": {
+            "title": "ENSEMBL ProteinID",
+            "description": "ENSEMBL ProteinID of official nomenclature approved protein. The ProteinID does not include the current version number suffix.",
+            "type": "string",
+            "pattern": "^ENSP\\d{11}.?\\d*?$",
+            "submissionExample": {
+                "appscript": "ENSP00000001146.2",
+                "igvf_utils": "ENSP00000001146.2"
+            }
+        },
+        "dbxrefs": {
+            "title": "External Resources",
+            "description": "Unique identifiers from the hORFeome database",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "External Identifier",
+                "description": "A unique identifier from hORFeome database.",
+                "comment": "Submit as database_name:id. See changelog for potential databases.",
+                "type": "string",
+                "pattern": "^(hORFeome:[1-9][0-9]*)$"
+            },
+            "submissionExample": {
+                "appscript": "[\"hORFeome:8945/\", \"hORFeome:8946\"]",
+                "igvf_utils": "hORFeome:8945, hORFeome:8946"
+            }
+        },
+        "pct_identical_protein": {
+            "title": "Percentage of Identical Matches to Ensembl Protein ID",
+            "description": "The Percentage of identical matches to Ensembl protein ID.",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100,
+            "submissionExample": {
+                "appscript": 34,
+                "igvf_utils": 34
+            }
+        },
+        "pct_coverage_protein": {
+            "title": "Percentage of ORF Covered by Ensembl Protein",
+            "description": "The percentage of ORF covered by Ensembl protein.",
+            "comment": " The percentage of the alignment between the ORF and it’s corresponding protein id over the length of the protein id.",
```

Comment:
Extra space at beginning, "its" instead of "it's"

Reply:
done